### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ghostlandr**

### CODE REVIEW

**Comment by ghostlandr** [comment]
Take it into consideration this time.

**Comment by evansalter** [comment]
Change the header on the page as well and I'll merge it :)

**Comment by ghostlandr** [comment]
Siiiick.

I'll make it `Scrobblin'` as well.

File: index.html [hunk 1]
```diff
@@ -27,4 +27,4 @@
         <script src="static/js/app.js"></script>
 
     </body>
-</html>
```
Comment (evansalter) [inline index.html:30]: Damn VSCode :/

**Review by evansalter — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -20,11 +17,15 @@
         <script type="text/javascript" src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
         <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.8/js/materialize.min.js"></script>
 
+        <script src="https://unpkg.com/vue/dist/vue.js"></script>
+        <script src="https://unpkg.com/vue-resource@1.0.3/dist/vue-resource.min.js"></script>
+        <script src="https://unpkg.com/vue-router/dist/vue-router.js"></script>
```
Comment (evansalter) [inline index.html:22]: 👏  good call

File: index.html (hunk 1 above)
Comment (ghostlandr) [reply index.html:30]: Hehe

**Pull request merged**